src/components/modals/SendFormModal.vue

```diff
@@ -153,27 +154,35 @@ const SendFormModal = defineComponent({
     rate: { type: Object as PropType<WalletRate>, required: true },
     balance: { type: Array as PropType<Coin[]>, required: true },
   },
-  setup: function () {
-    const form = useForm({
+  setup: function (props) {
+    const fee = ref(0)
+    const isLoading = ref(false)
+    const onSubmit = dialogs.getHandler('onSubmit')
+    const onClose = preventIf(dialogs.getHandler('onClose'), isLoading)
+    const sendAsset = ref('loki')
+    const selectedBalance = computed(() => {
+      const balance = props.balance.find((item) => {
+        return item.denom === sendAsset.value ? true : false
```

Comment:
return item.denom === sendAsset.value will be enough,
=== operator already returns true or false